clients/src/main/java/org/apache/kafka/common/security/authenticator/LoginManager.java

```diff
@@ -61,7 +61,7 @@ private LoginManager(LoginType loginType, boolean hasKerberos, Map<String, ?> co
      *                  (i.e. consumer and producer)
      * @param configs configuration as key/value pairs
      */
-    public static final LoginManager acquireLoginManager(LoginType loginType, boolean hasKerberos, Map<String, ?> configs) throws IOException, LoginException {
+    public static LoginManager acquireLoginManager(LoginType loginType, boolean hasKerberos, Map<String, ?> configs) throws IOException, LoginException {
```

Comment:
Why was this removed? Maybe an incorrect suggestion by an automated tool? See https://netbeans.org/bugzilla/show_bug.cgi?id=230827 for example.